File: java-samples/corda5-negotiation-cordapp/workflows/src/main/java/com/r3/developers/samples/negotiation/workflows/accept/AcceptFlowArgs.java
package com.r3.developers.samples.negotiation.workflows.accept;

import net.corda.v5.base.annotations.CordaSerializable;
import net.corda.v5.base.types.MemberX500Name;

import java.util.UUID;

@CordaSerializable
public class AcceptFlowArgs {

    private UUID proposalID;
    private MemberX500Name acceptor;

    public AcceptFlowArgs() {
    }

    public AcceptFlowArgs(UUID proposalId, MemberX500Name acceptor) {
        this.proposalID = proposalId;
        this.acceptor = acceptor;
    }

    public UUID getProposalID() {
        return proposalID;
    }

    public MemberX500Name getAcceptor() {
        return acceptor;
    }
}

File: kotlin-samples/corda5-negotiation/workflows/src/main/kotlin/com/r3/developers/samples/negotiation/workflows/Accept/AcceptFlowArgs.kt
package com.r3.developers.samples.negotiation.workflows.Accept

import net.corda.v5.base.annotations.CordaSerializable
import net.corda.v5.base.types.MemberX500Name
import java.util.*

@CordaSerializable
data class AcceptFlowArgs(
    var proposalID: UUID,
    var acceptor: MemberX500Name
)
